lib/NetworkManager.js

```diff
@@ -773,9 +776,12 @@ const STATUS_TEXTS = {
   '422': 'Unprocessable Entity',
   '423': 'Locked',
   '424': 'Failed Dependency',
+  '425': 'Too Early',
   '426': 'Upgrade Required',
+  '427': 'Unassigned',
   '428': 'Precondition Required',
   '429': 'Too Many Requests',
+  '430': 'Unassigned',
```

Comment:
unassigned just means unassigned, its not a specific code.

Reply:
indeed!